WasatchNET/MockSpectrometer.cs

```diff
@@ -0,0 +1,546 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
+using System.Threading;
+using LibUsbDotNet;
+using LibUsbDotNet.Main;
+
+namespace WasatchNET
+{
+    public class MockSpectrometer : Spectrometer

```

Comment:
A class comment would be good here.  Maybe a long one.

Reply:
done